includes/REST_API/Stories_Controller.php

```diff
@@ -122,6 +122,27 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Updates a single post.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function update_item( $request ) {
+		$response = parent::update_item( $request );
+		if ( ! is_wp_error( $response ) ) {
+			// If publisher logo is set, let's assign that.
+			$publisher_logo_id = $request->get_param( 'publisher_logo' );
+			if ( $publisher_logo_id ) {
+				// @todo This option can keep track of all available publisher logo IDs in the future, thus the array.
+				$publisher_logo_settings           = get_option( self::PUBLISHER_SETTINGS_OPTION, [] );
+				$publisher_logo_settings['active'] = $publisher_logo_id;
+				update_option( self::PUBLISHER_SETTINGS_OPTION, $publisher_logo_settings, false );
```

Comment:
So now there's an option named `web_stories_publisher_settings` with the following content:

```
[
  'active' => 123
]
```

I'd either change the name to something like `web_stories_publisher_logos` or change the data structure, e.g.:

```
[
  'publisherLogos': {
    'active' => 123
  }
]
```

See also #754. We don't have to worry about that too much.

Reply:
Okay, will change to `web_stories_publisher_logos`.

The idea is to have it something like for when there are multiple logos later.
```
[
 'active' => 123,
 'ids' => [1,2,3,4],
]
```

Reply:
Changed in 1b012f3